testsuite/config/cucumber.yml

```diff
@@ -33,7 +33,7 @@ configuration_channels: --tags @scope_configuration_channels
 content_staging: --tags @scope_content_staging
 proxy: --tags @scope_proxy
 traditional_client: --tags @scope_traditional_client
-xmlrpc: --tags @scope_xmlrpc
+api: --tags @scope_api
```

Comment:
At least for history of discussions.
I discussed with Eric that I would prefer continue covering all the xmlrpc_xxx features we had in all our CI pipelines.
And add a basic set of new features, specific for http api.

So, my proposal was to use a first scenario that will set which kind of api it will be instantiated, depending the kind of test to pass.

So, about scopes, we will have two scopes: scope_http_api and scope_xmlrpc_api.

As the way Eric preferred to design it, the only feasible way to use one or the other is by changing a line of code (or a variable), but it will always affect all the testsuite.

Therefore, if we want to test HTTP API we are forced to lose all the test coverage of XMLRPC API tests.

Reply:
No, we can instanciate 2 variables, one for HTTP and one for XML-RPC. Then you can create a step to choose the API to use. It works, I have tested it. We can switch to a mix of HTTP and XML-RPC whenever we want. Or to a choice for sumaform.

The fact is that I do want to test intensively the new HTTP API, at least during some time. That means running **all** the tests with this API on at least one branch during some time. That's the reason why I don't want this "mix", at least at the beginning.

The solution for now will be to test Uyuni with HTTP and Head with XML-RPC. As I said, we can change that any time, But for now, please, let's have a few full runs with HTTP, so we don't miss any bug.

Besides that, a remark: the current `@scope_api` affects only 4 features. There's a whole lot of other features, of scenarios, and even of individual steps, that use the API in some manner.